cpp/src/join/mixed_join_common_utils.cuh

```diff
@@ -28,6 +29,14 @@
 namespace cudf {
 namespace detail {
 
+using row_hash_experimental =
+  cudf::experimental::row::hash::device_row_hasher<default_hash, cudf::nullate::DYNAMIC>;
+
+// // This alias is used by mixed_joins, which support only non-nested types
+using row_equality_experimental =
```

Comment:
Don't use the name with `_experimental` in it.

Reply:
I prefer that name since there's a type-alias for the legacy comparator called `row_equality` [here](https://github.com/rapidsai/cudf/blob/2477695b18b742e776db9f47604749a3e7792539/cpp/src/join/join_common_utils.hpp#L65). When we refactor to remove the experimental namespace, I will remove this as well.

Reply:
So why don't just use a better name now for it, and use something like `_legacy` name for the old one, so you just need to remove the old one while don't have to change name for the new one in the future?